src/main/java/sample/views/LearnWindow.java

```diff
@@ -51,6 +54,13 @@ public class LearnWindow extends Application {
     private final GridPane keyPane = new GridPane();
     private final Button resetButton = new Button("RESET");
 
+    private final Button homeButton = new Button("Home");
+
+    private Text text = new Text("");
+
+    private ArrayList <Button> button_sequence = new ArrayList<>();
+
+    private String text_diplay_when_clicked = "";
```

Comment:
This variable is not really necessary. The places where it is used can make use can generate this text inline.